testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveTransactionalTable.java

```diff
@@ -1836,6 +1837,33 @@ public void testDoubleUpdateAndThenReadFromHive()
         });
     }
 
+    @Test(groups = HIVE_TRANSACTIONAL)
+    public void testDeleteWithOriginalFiles()
+    {
+        withTemporaryTable("test_delete_with_no_predicate_on_ctas", true, false, NONE, tableName -> {
+            onTrino().executeQuery("SET SESSION scale_writers = true"); //these 3 properties are necessary to make sure there is more than 1 original file created
+            onTrino().executeQuery("SET SESSION writer_min_size = '4kB'");
+            onTrino().executeQuery("SET SESSION task_writer_count = 2");
+            onTrino().executeQuery(format(
+                    "CREATE TABLE %s WITH (transactional = true) AS SELECT * FROM tpch.sf1000.orders LIMIT 100000", tableName));
+
+            verify(onTrino().executeQuery(format("SELECT DISTINCT \"$path\" FROM %s", tableName)).getRowsCount() >= 2,
+                    "There should be at least 2 files");
+            validateFileIsDirectlyUnderTableLocation(tableName);
+
+            onTrino().executeQuery(format("DELETE FROM %s", tableName));
+            verifySelectForTrinoAndHive(format("SELECT COUNT(*) FROM %s", tableName), "TRUE", row(0));
+        });
+    }
+
+    private void validateFileIsDirectlyUnderTableLocation(String tableName)
+    {
+        onTrino().executeQuery(format("SELECT DISTINCT regexp_replace(\"$path\", '%s', '') FROM %s", new StringBuilder("/[^/]*$"), tableName))
+                .column(1)
+                .forEach(path -> verify(path.toString().toLowerCase(Locale.ROOT).endsWith(tableName.toLowerCase(Locale.ROOT)),
```

Comment:
we typically lowerCase with Locale.ENGLISH (imported statically)

here however, i think no lowercasing should be needed, as both table name and its path should already be lowercase.

Reply:
It fails without it, for some reason there is `NONE` in the `tableName` and `none` in the `path`

Reply:
Will use English, this was generated by IDE ;)

Reply:
you're right 

NONE comes from `bucketingType.name()` 
https://github.com/trinodb/trino/blob/a72a739861197d08fae77e8d20054473de6142fb/testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveTransactionalTable.java#L2003

you can add lowercase in `tableName` method.